README.md

```diff
@@ -20,6 +20,31 @@ kashmir
 [npm-image]: https://badge.fury.io/js/generator-kashmir.svg
 [npm-url]: https://npmjs.org/package/generator-kashmir
 
+
+## Build commands
+---
+
+### Compile theme
+
+In order to compile the assets of the theme, this also compiles sass inside components,
+
+```bash
+gulp
+```
+
+### Global css
```

Comment:
Do we need a section for a CSS file?

Reply:
I think it is good to have this as how CSS is hooked with the theme.

Reply:
@skippednote Yes, I will add it for the TS.